docs/en_US/Tuner/BuiltinTuner.rst

```diff
@@ -574,6 +575,41 @@ Population Based Training (PBT) bridges and extends parallel search methods and
 
 Note that, to use this tuner, your trial code should be modified accordingly, please refer to `the document of PBTTuner <./PBTTuner.rst>`__ for details.
 
+DNGO Tuner
+^^^^^^^^^^
+
+..
+
+   Built-in Tuner Name: **DNGOTuner**
+
+DNGO advisor requires `pybnn`, which can be installed with the following command.
+
+.. code-block:: bash
+
+   pip install nni[DNGO]
+
+**Suggested scenario**
+
+Applicable to large scale hyperparameter optimization. Bayesian optimization that rapidly finds competitive models on benchmark object recognition tasks using convolutional networks, and image caption generation using neural language models.
+
+**classArgs requirements:**
+
+
+* **optimize_mode** (*'maximize' or 'minimize'*\ ) - If 'maximize', the tuner will target to maximize metrics. If 'minimize', the tuner will target to minimize metrics.
+* **sample_size** (*int, default = 1000*) - Number of samples to select in each iteration. The best one will be picked from the sample.
```

Comment:
-> The best one will be picked from the sample as the next trial.

Reply:
updated